teamA/jinwoo/app.js

```diff
@@ -33,6 +33,143 @@ app.get("/ping", (req, res) => {
   res.status(200).json({ message : "pong" });
 })
 
+// 전체 게시글 조회하기
+app.get("/posts", async (req, res) => {
+  await mysqlDataSource.query(
+    `SELECT
+        usersWesta.id as userId,
+        usersWesta.profile_image as userProfileImage,
+        postsWesta.id as postingId,
+        postsWesta.title as postingTitle,
+        postsWesta.content as postingContent
+      FROM usersWesta
+      INNER JOIN postsWesta
+      ON postsWesta.user_id = usersWesta.id
+      `
+    ,(err, rows) => {
+        res.status(200).json(rows);
+      })
+});
+
+// 유저의 게시글 조회하기
+app.get("/posts/:userId", async (req, res, next) => {
+  const userId = req.params.userId;
+
+  const userInfo = await mysqlDataSource.query(
+    `SELECT
+        usersWesta.id as userId,
+        usersWesta.profile_image
+      FROM usersWesta
+      WHERE id = ${userId};
+    `);
+    
+    const posting = await mysqlDataSource.query(
+      `SELECT
+        postsWesta.id as postingId,
+        postsWesta.title as postingTitle,
+        postsWesta.content as postingContent
+      FROM postsWesta
+      INNER JOIN usersWesta
+      ON usersWesta.id = postsWesta.user_id
+      `
+    );
+      userInfo[0].postings = posting;
+      res.status(200).json({ "data" : userInfo[0] });
+    });
+
+// 유저 회원가입 하기
+app.post("/signup", async (req, res, next) => {
+  const { name, email, profileImage, password } = req.body
+  await mysqlDataSource.query(
+    `INSERT INTO usersWesta(
+      name,
+      email,
+      profile_image,
+      password
+    )
+    VALUES (?, ?, ?, ?)
+    `,
+    [ name, email, profileImage, password ]
+  );
+  res.status(201).json({ message : "Successfully created" });
+})
+
+// 게시물 등록하기
+app.post("/postup", async (req, res) => {
```

Comment:
```suggestion
app.post("/posts", async (req, res) => {
```
해당 내용도 RESTful API 세션을 들은 후 다시한번 돌아봐 주세요 :)